### DISCUSSION TITLE
Can't connect using kerberos ticket

### DISCUSSION THREAD

**Discussion by Whidix** [notroj/cadaver#47] [General]
**Describe the bug**
I'm running alfresco using sso, I want to mount the webdav in my system but it ask me for credentials. Since I'm using yubikeys and certificates to login against freeipa with kerberos, my user don't have them. 

**Environment**
 - cadaver version: [0.23.3]
 - neon version: [0.30.2]
 - OS: [RHEL 8]
 - SSL library version: [OpenSSL 1.1.1

**Comment by notroj** [comment]
That version of `neon` supports `Negotiate` (Kerberos over HTTP) auth but it can be tricky to configure. cadaver will only support `Negotiate` over `https://` connections - if you're using that, please post the output of
```
$ cadaver
dav:!> set debug http,httpauth
dav:!> open https://example.com/path
```
